Python/getargs.c

```diff
@@ -485,11 +494,19 @@ converttuple(PyObject *arg, const char **p_format, va_list *p_va, int flags,
             n++;
     }
 
+    if (arg == Py_None && nullable) {
+        const char *msg = skipitem(p_format, p_va, flags);
+        if (msg != NULL) {
+            levels[0] = 0;
+        }
+        return msg;
+    }
     if (!PySequence_Check(arg) || PyBytes_Check(arg)) {
         levels[0] = 0;
         PyOS_snprintf(msgbuf, bufsize,
-                      "must be %d-item sequence, not %.50s",
+                      "must be %d-item sequence%s, not %.50s",
```

Comment:
```suggestion
                  "must be %d-item sequence%s, not %.50s",
```

Either align "must" with msgbuf, or with `n` below. The call was a bit hard to read.